src/ip_lambert_conf_grid_mod.F90

```diff
@@ -17,8 +22,8 @@ module ip_lambert_conf_grid_mod
   public :: ip_lambert_conf_grid
 
   type, extends(ip_grid) :: ip_lambert_conf_grid
-     real :: rlat1 !< ???
-     real :: rlon1 !< ???
+     real :: rlat1 !< La1― latitude of first grid point. GRIB2, Section 3.30, octet 39-42.
+     real :: rlon1 !< Lo1― longitude of first grid point. GRIB2, Section 3.30, octet 43-46.
      real :: rlati1 !< ???
```

Comment:
rlati1 - First latitude from the pole at which the secant cone cuts the sphere. GRIB2, Section 3, octets 66-69.
rlati2 - Second latitude from the pole at which the scant cone cuts the sphere. GRIB2, Section 3, octets 70-73.
orient - Longitude of meridian parallel to y-axis along which latitude increases at the latitude increases. GRIB2, Section 3, octets 52-55.

Reply:
dxs - x-direction grid length adjusted for scan mode. GRIB2, Section 3, octets 56-59.
dys - y-direction grid length adjusted for scan model. GRIB2, Section 3, octets 60-63.

Reply:
irot - vector rotation flag. When "1", vectors are grid relative. When "0", vectors are earth relative. GRIB2, Section 3, octet 55.

Reply:
Thanks! I have filled those in.

Below them in the code there are these:

  INTEGER :: IROT !< ???
  REAL :: AN !< ???
  REAL :: DXS !< ???
  REAL :: DYS !< ???
  REAL :: H !< ???
  REAL :: RERTH !< ???

Reply:
AN - Cone factor
RERTH - Radius of the earth. GRIB2, Section 3, octets 15-30.

Note: AN is not an element in the GRIB2 template. So, there is no octet associated with it.

Also, a good reference for all the map projection equations used by IPLIB is this: https://doi.org/10.3133/pp1395
You might consider adding that to the wiki page or one of the readme files.